test/test_operations.py

```diff
@@ -172,6 +172,8 @@ def _xla_run(model, input, device='TPU'):
 
 def _forward_passes(graph):
     torch._C._jit_pass_canonicalize_ops(graph)
+    torch_xla._C._jit_pass_set_mat_mul_output_shape(graph)
+    torch_xla._C._jit_pass_insert_explicit_expand(graph)
```

Comment:
Remember these moved into the test/xla_model_running_utils.py file.

Reply:
Yup, fortunately it shows as a merge conflict so I can't forget about it.